datahub-actions/src/datahub_actions/action/action_registry.py

```diff
@@ -16,7 +16,11 @@
 
 from datahub_actions.action.action import Action
 from datahub_actions.plugin.action.hello_world.hello_world import HelloWorldAction
+from datahub_actions.plugin.action.metadata_change_sync.metadata_change_sync import (
+    MetadataChangeSyncAction,
+)
 
 action_registry = PluginRegistry[Action]()
 action_registry.register_from_entrypoint("datahub_actions.action.plugins")
 action_registry.register("hello_world", HelloWorldAction)
+action_registry.register("metadata_change_sync", MetadataChangeSyncAction)
```

Comment:
we usually put these in setup.py instead, which enables dynamic discovery and lets us lazy load the imports